baseapp/baseapp_payments/serializers.py

```diff
@@ -13,28 +13,75 @@
 
 
 class StripeSubscriptionSerializer(serializers.Serializer):
-    remote_customer_id = serializers.CharField()
-    price_id = serializers.CharField()
-    remote_subscription_id = serializers.ReadOnlyField()
+
+    remote_customer_id = serializers.CharField(help_text="Stripe customer ID")
+    price_id = serializers.CharField(help_text="Stripe price ID")
+    allow_incomplete = serializers.BooleanField(
+        default=False,
+    )
+    payment_method_id = serializers.CharField(
+        required=False,
+    )
+    billing_details = serializers.DictField(
+        required=False,
+    )
+
+    remote_subscription_id = serializers.CharField(read_only=True)
+    client_secret = serializers.CharField(read_only=True)
+    invoice_id = serializers.CharField(read_only=True)
 
     def validate(self, data):
+        customer_id = data["remote_customer_id"]
+        price_id = data["price_id"]
+
+        subscriptions = StripeService().list_subscriptions(customer_id, status="all")
```

Comment:
I think there's a logical issue here...  I don't think it matters if a person is subscribed to two of the same price (I mean this shouldn't happen either), I think we want to make sure that the user isn't subscribed to the same **product** twice.  The way you have it here, technically a user could be subscribed to the same product, just on two different prices, if that makes sense